apps/web/components/login.tsx

```diff
@@ -123,40 +124,95 @@ export default function Login(props: any) {
                   className="group relative flex w-full">
                   <span className="absolute inset-y-0 left-0 flex items-center pl-3">
                     <LockClosedIcon
-                      className="text-neon-700 group-hover:text-neon-dark-700 h-5 w-5 disabled:disabled:bg-gray-600 disabled:group-hover:bg-gray-600 duration-200"
+                      className="text-neon-700 group-hover:text-neon-dark-700 h-5 w-5 duration-200 disabled:disabled:bg-gray-600 disabled:group-hover:bg-gray-600"
                       aria-hidden="true"
                     />
                   </span>
                   Sign in
                 </Button>
               </div>
               <div>
-                <div className="relative">
+                <div className="relative my-2">
                   <div className="absolute inset-0 flex items-center" aria-hidden="true">
                     <div className="w-full border-t border-gray-300" />
                   </div>
-                  <div className="relative flex justify-center"></div>
+                  <div className="relative flex justify-center text-sm">
+                    <span className="bg-gray-100 px-2 text-gray-500">Or continue with</span>
```

Comment:
This can probably just be `or` and we can apply an `uppercase` class since the below provider says "Sign in with Google"